package.json

```diff
@@ -1,102 +1,126 @@
 {
-    "name": "@cpanel/api",
-    "version": "5.0.0",
-    "description": "cPanel API JavaScript and TypeScript interface libraries. This library provides a set of classes for calling cPanel WHM API 1 and UAPI calls. The classes hide much of the complexity of these APIs behind classes the abstract the underlying variances between these API systems. Users of this library can focus on what they want to accomplish rather the having to learn the various complexities of the underlying wire formats for each of the cPanel Products APIs",
-    "main": "dist/index.js",
-    "types": "dist/index.d.ts",
-    "sideEffects": false,
-    "files": [
-        "*.d.ts",
-        "*.js",
-        "*.map.js",
-        "**/*.d.ts",
-        "**/*.js",
-        "**/*.map.js",
-        "documentation/**/*"
-    ],
-    "scripts": {
-        "build:prod": "npm run clean && npm run build:ts && npm run webpack:prod",
-        "build:dev": "npm run clean && npm run build:ts-dev && npm run webpack:dev",
-        "build:ts": "tsc -p tsconfig.prod.json",
-        "build:ts-dev": "tsc -p tsconfig.dev.json",
-        "build:watch": "tsc -w -p tsconfig.json",
-        "webpack:dev": "webpack --env dev",
-        "webpack:prod": "webpack --env prod",
-        "clean": "rm -rf dist",
-        "build-docs": "compodoc -p tsconfig.dev.json -d ./docs",
-        "serve-docs": "compodoc -p tsconfig.dev.json -d ./docs -r 4214 -s",
-        "test": "npm run build:dev && jasmine --config=jasmine.json --reporter=jasmine-console-reporter",
-        "test:cover": "nyc -r lcov -x '**/*.spec.ts' jasmine --config=jasmine.json && nyc report",
-        "test:watch": "npm run build:watch && onchange -i -v -d 100 dist -- jasmine --config=jasmine.json --reporter=jasmine-console-reporter",
-        "ts-tidy": "npx eslint -c .ts-eslint-tidy-rc.json 'src/**/*.ts' --fix",
-        "prepublish": "npm run build:prod"
+  "name": "@cpanel/api",
+  "version": "5.0.1",
```

Comment:
As we discussed in Slack, this change should be invisible to consumers because the `package.json` points them to the correct folder based on how they are consuming the library.